tools/asset-sync/assets.ps1

```diff
@@ -0,0 +1,521 @@
+Set-StrictMode -Version 3
+
+# See https://stackoverflow.com/a/14440066 for a succinct explanation. We will be using
+# this methodology for any global variables that can be considered constants
```

Comment:
I'm not sure what this comment/stack overflow link applies to?

Reply:
Legit not certain.